charts/base/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.66
```

Comment:
the change is branching one and for setups were we use initContainers with old/non-multi form upgrading to this new version will break things,

so at least minor version change needed with note in readme, default values.yam,

also I thing we should have already some example in README.md, so please update example to include old and new forms by specifying from which version we have the new form,
and I think we also should have some commented initContainers block in default values.yaml, which also need to be changed to new format